Makefile

```diff
@@ -29,14 +34,24 @@ df-demo:
 	go build -o bin/demoserver ./cmd/demoserver
 	cp ./bin/demoserver ./ansible/roles/common/files/demoserver.bin
 
-unit_test:
+unit_test: check_failpoint_ctl
 	mkdir -p "$(TEST_DIR)"
+	$(FAILPOINT_ENABLE)
 	$(GOTEST) -cover -covermode=atomic -coverprofile="$(TEST_DIR)/cov.unit.out" $(PACKAGES)
+	$(FAILPOINT_DISABLE)
```

Comment:
Better to use `command || { $(FAILPOINT_DISABLE); exit 1; }` in case that `command` fails and failpoint disable is not executed, which leaves the code base untidy.

ref: https://github.com/pingcap/tiflow/blob/02baae1916cedaec826c635949b8dd094bb2313d/Makefile#L119-L120